src/odemis/acq/test/fastem_test.py

```diff
@@ -108,16 +109,18 @@ def test_overview_acquisition(self):
         self.assertGreaterEqual(bbox[3], scintillator5_area[3])  # Top
 
 
+@unittest.skip("Can't run ROA tests until all simulator dependencies have been installed on the test VM.")
```

Comment:
What is the reason to not do it now?

Reply:
Typically, I think we shouldn't care about the "test VM". If the test cases are expected to pass (and I hope they are!), then they should be run. If the "test VM" has some limitation, so be it, but we just have to interpret the results from the test VM, not skip the tests on every computer.

Reply:
Ok, then I'll remove it.

> What is the reason to not do it now?
It will take some time install the simulator and update the test script to start the simulator and xtadapter. As there are already so many test cases that fail, I discussed with @Kleijwegt to first fix the rest of the test cases before adding other things.